README.md

```diff
@@ -1,5 +1,36 @@
 # moxiedoc
 
+## Introduction
+
+This project maintains Moxiedoc, a tool used to build API reference documentation. If you have any modifications you wish to contribute, fork this project, make the changes and submit a pull request. You will need to sign the contributors license agreement, which will be emailed to you upon creating the pull request.
+
+## Using Moxiedoc
+
+To create API reference documentation from a development version of moxiedoc, run:
+
+```
+yarn build
+node ./dist/lib/cli.js PATH/TO/API_FILE_FOLDER
+```
+
+## Moxiedoc Options
+
+Moxiedoc provides the following options to customise the format of the output documentation:
+
+```
+-o --out <path>: location of output files, default: 'tmp/out.zip'
+-t --template <template>: documentation type: default: 'cli'; 'antora', 'github', 'moxiewiki', 'singlehtml', 'tinymcenext', 'xml'
+-s --structure <type>: default: 'default'; 'legacy'
```

Comment:
I forgot to update the PR description from the first draft, sorry. It is now just either no value (default), `default`, or `legacy`. This is since each template has a different structure so only default seems appropriate.

Reply:
Updated the PR description now